spec/dummy/config/application.rb

```diff
@@ -26,6 +26,9 @@ class Application < Rails::Application
       config.load_defaults ENV["RAILS_VERSION"] || 7.0
     end
 
+    config.i18n.default_locale = :en
+    config.i18n.available_locales = [:en, :de, :kl]
```

Comment:
This breaks the test suite. Maybe we do not add the gem now? Or adjust the specs, but this is much more involved I guess

Reply:
I moved it into the development.rb to not break the tests